junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/TestFactoryTestDescriptor.java

```diff
@@ -69,6 +70,18 @@ public boolean isLeaf() {
 	}
 
 	@Override
+	public JupiterEngineExecutionContext execute(JupiterEngineExecutionContext context,
```

Comment:
Unless I'm mistaken, lifecycle callbacks currently apply to `@TestFactory` methods.

So it appears that you have removed this functionality, which some people may depend on.